### ISSUE TITLE
Beginner troubleshooting

### ISSUE THREAD

**Issue by jandersonIEG** [VUKOZ-OEL/3d-forest-classic#19]
Hi there! I am really interested in the 3dForest project, and I am hoping for some help. 

Firstly, can you help me understand the transformation matrix, is it just in the projection that the LAS file it using? in my case NAD83, zone 8N. 

This issue may be related to my problems with the first, but when I import an las file, the command prompt says the import is complete, but no point cloud appears. Do you have any ideas what I may be doing wrong? The las file is quite large. 

Cheers, and thanks for your hard work on this. 
Jeff

**Comment by janekT** [comment]
HI Jeff, 
Transformation matrix deal with long cordinates to display in proper format. Coordinates like X:5820882.9786, Y:703625.1702
will result in 5820882, Y:703625.1 thanks to float precision. So without transformation matrix you lost some precision data. When you apply matrix  that shift data close to zero  ( like: -5820880, -703625)  you are not loosing precision in detail and when exporting data from 3D forest, matrix is applied again to get correct coordinates.

**Comment by janekT** [comment]
And for the second problem if import work and your las file appear in left column,try to press "r" or one of icons for views (top, side view, ..) laso check if your layer is selected for visualization in left column.

**Comment by tildigg** [comment]
Hi there,

Super excited to get this application working. I am also having this issue.

When i import my large las file the point cloud doesnt show. If i import a re sampled point cloud it does show but the points are fitted into straught lines. Is this an issue with the transofrmation matrix? And if so how do i calculate the tranformation matrix of my point cloud?

Many thanks,
Mathilda

**Comment by janekT** [comment]
yes it seems to be problem with matrix. If you have lastools installed, try to open your file in i.e. lastile and in lower left corner you will see bounding box of your data. Then try to put into matrix x,y,z values with negative sign  and import las file. The result should be close to zero. formula is: `las_coordinate + transf.matrix_value = 0`

as example: 

las file coordinates: 

> X: 45621687.488631
> Y:-123399269.47899
> Z: 453.406

then in transformation matrix should be something like this:

> X: -45621687.0
> Y: 123399269.0
> Z: 0

Hope it helps
Jan

**Comment by msimmar26** [comment]
Hello! 
I'm also a beginner with the software. For now I'm using the example data provided on the website, however I don't know which transformation matrix I should use for those datasets. There is like a default existing matrix "WindRiver" on the software, should I use that one or create my own? 
Thaks!
Marina
